contracts/test/v0.8/dev/KeeperRegistryDev.test.ts

```diff
@@ -2285,6 +2286,29 @@ describe('KeeperRegistryDev', () => {
         )
         expect((await registry2.getUpkeep(id)).checkData).to.equal(randomBytes)
       })
+
+      it('migrates a paused upkeep', async () => {
+        expect((await registry.getUpkeep(id)).balance).to.equal(toWei('100'))
+        expect((await registry.getUpkeep(id)).checkData).to.equal(randomBytes)
+        expect((await registry.getState()).state.numUpkeeps).to.equal(1)
+        await registry.connect(admin).pauseUpkeep(id)
+        // verify the upkeep is paused
+        expect((await registry.getUpkeep(id)).paused).to.equal(true)
+        // migrate
+        await registry.connect(admin).migrateUpkeeps([id], registry2.address)
+        expect((await registry.getState()).state.numUpkeeps).to.equal(0)
+        expect((await registry2.getState()).state.numUpkeeps).to.equal(1)
+        expect((await registry.getUpkeep(id)).balance).to.equal(0)
+        expect((await registry2.getUpkeep(id)).balance).to.equal(toWei('100'))
+        expect((await registry.getUpkeep(id)).checkData).to.equal('0x')
+        expect((await registry2.getUpkeep(id)).checkData).to.equal(randomBytes)
```

Comment:
why randomBytes? Should be 0x?

Reply:
i think this upkeep is created with checkData = randomBytes.
so after migration, the old upkeep's checkData is deleted and the new upkeep has randomBytes as checkData

Reply:
oh yeah i misunderstood before